dvc/output.py

```diff
@@ -185,6 +188,7 @@ def load_from_pipeline(stage, data, typ="outs"):
                 Output.PARAM_CACHE,
                 Output.PARAM_PERSIST,
                 Output.PARAM_CHECKPOINT,
+                Output.PARAM_REMOTE,
```

Comment:
Would there be a need for a stage-level remote?

Reply:
Maybe in the future, but not for now.

Reply:
okay, I was just wondering if it makes sense in the stage-level, or should there be a different keyword there?

Reply:
`push_to: <remote>` or something.

Reply:
It might be that we'll need to support pushing/pulling granularly in the future, but for now this is a symmetrical. Interesting to note is that our imports are kinda like `pull: orig_remote, push: false`, since they are pulled from external repo remote but not pushed anywhere by default. So that might be a better way to handle https://github.com/iterative/dvc/pull/4581 (e.g. `push: mybackup` or just `true` for default remote). But that's beyond the scope for this PR. Or do you see where `remote: ` might get problematic in the future?